Sources/Frontend/src/views/BoardPage.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <div>
+    <h1>Success</h1>
+  </div>
+</template>
+
+<script setup lang="ts">
+  import * as THREE from 'three';
+
+  class GameBoard {
```

Comment:
Lets move this class to "Rendering" (or something like this) folder, and initialize with canvas selected from template
This may not work in setup part because html may be inserted to dom later, check docs